src/transfer/transfer.page.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useEffect } from "react";
+import {
+  Divider,
+  Grid,
+  CircularProgress,
+  Paper,
+  Box,
+  TableContainer,
+  Table,
+  TableBody,
+  TableHead,
+  TableRow,
+  Button,
+  TableFooter,
+  TablePagination,
+} from "@mui/material";
+import { Add } from "@mui/icons-material";
+import { users } from "../data/user";
+import { StyledTableCell, StyledTableRow } from "./transfer.styles";
+import { useNavigate } from "react-router-dom";
+
+export const Transfer = () => {
+  const [page, setPage] = React.useState(0);
+  const [rowsPerPage, setRowsPerPage] = React.useState(5);
+  const handleChangePage = (
+    event: React.MouseEvent<HTMLButtonElement> | null,
+    newPage: number
+  ) => {
+    setPage(newPage);
+  };
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  const [beneficiaries, setBeneficiaries] = useState([
+    {
+      accountNumber: "",
+      firstName: "",
+      lastName: "",
+      bankName: "",
+    },
+  ]);
+  const [loading, setLoading] = useState(true);
+  const navigate = useNavigate();
+  useEffect(() => {
+    const timeout = setTimeout(() => {
+      setLoading(false);
+      setBeneficiaries(users[0].beneficiaries.reverse());
```

Comment:
Not sure why you are using reverse here?

Reply:
To display the latest added beneficiaries on top, this step is usually done on backend on a mongo aggregation pipeline but since I didn't have that I settled for this for now.